backend/Dockerfile

```diff
@@ -14,11 +14,21 @@ COPY . .
 RUN apk add --update gcc musl-dev
 RUN go build -o /bin/apiserver backend/src/apiserver/*.go
 
-FROM python:3.5.0-slim as compiler
+FROM python:3.5 as compiler
+
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y -q default-jdk wget 
+
+RUN pip3 install setuptools==40.5.0
```

Comment:
Is this no auto-installed? Do we need the specific version?

Reply:
Yes per @gaoning777 's investigation.